robot/client/client.go

```diff
@@ -131,7 +131,7 @@ func isClosedPipeError(err error) bool {
 }
 
 func (rc *RobotClient) notConnectedToRemoteError() error {
-	return errors.Errorf("not connected to remote robot at %s", rc.address)
```

Comment:
This usage adds a stack trace to the error. I found them to be an eyesore and not at all helpful.